src/Console/Application.php

```diff
@@ -188,6 +188,8 @@ private function logRunningWithDebugger(): void
             $this->consoleOutput->logRunningWithDebugger(\PHP_SAPI);
         } elseif (\extension_loaded('xdebug')) {
             $this->consoleOutput->logRunningWithDebugger('Xdebug');
+        } elseif (\extension_loaded('pcov')) {
+            $this->consoleOutput->logRunningWithDebugger('PCOV');
```

Comment:
I'm not entirely sure if we need this, but this may be a good idea in sake of verbosity.